# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/thebes/gfxFontEntry.h
# Commit: 673f9010ea75
# Full Hash: 673f9010ea75382463067acf3da66683f4eecae1
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2022-08-17 09:10:29
# Regressor Bug: 1740530
# File Overlap Count: 1
# Description:
#   Bug 1740530 - patch 3 - Replace GetColorGlyphLayers and the layer-rendering code in gfxFont with a zero-copy implementation in COLRFonts. r=gfx-reviewers,lsalzman
#   
#   Depends on D152037
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D152038
# ==============================================================================

diff -r 429601610f26 -r 673f9010ea75 gfx/thebes/gfxFontEntry.h
--- a/gfx/thebes/gfxFontEntry.h	Tue Aug 16 12:48:32 2022 +0000
+++ b/gfx/thebes/gfxFontEntry.h	Tue Aug 16 12:48:32 2022 +0000
@@ -58,9 +58,6 @@
 struct Face;
 struct Family;
 }  // namespace fontlist
-namespace gfx {
-struct DeviceColor;
-}
 }  // namespace mozilla
 
 typedef struct gr_face gr_face;
